filesystem.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	"github.com/bufbuild/connect-go"
+
```

Comment:
That definitely should be here 🙂

Reply:
Intersting no idea why my editor did that. I think I've configured it to behave properly now. But its reordered some other imports.